From 918724aa76982224437384d712c182c332fa5ef9 Mon Sep 17 00:00:00 2001
From: Dan Kenigsberg <danken@redhat.com>
Date: Wed, 21 Oct 2009 13:56:04 +0200
Subject: [PATCH] Do not log rotate very small logs

Without this, after few weeks without use, each defined domain grows a
tail of empty gzipped logs, instead of keeping just the last log of
interest.

* daemon/libvirtd.logrotate.in: only rotate when the log is over 100 KBytes

(cherry picked from commit b03fe2d0aefb57a096a102bf23375f0a167ca189)

Fedora-patch: libvirt-logrotate-avoid-compressing-small-logs.patch
---
 qemud/libvirtd.logrotate.in |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/qemud/libvirtd.logrotate.in b/qemud/libvirtd.logrotate.in
index 093651c..0c51fd3 100644
--- a/qemud/libvirtd.logrotate.in
+++ b/qemud/libvirtd.logrotate.in
@@ -5,4 +5,5 @@
         compress
         delaycompress
         copytruncate
+        minsize 100k
 }
-- 
1.6.2.5

